app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class AddressSearchController < ApplicationController
+      include RenderConditionConcern
+
+      check_or_render_not_found -> { InPersonConfig.enabled? }
```

Comment:
We can always add it later, but will we want a feature-flag specific for PO search? Are we comfortable shipping this code to production next week and receiving user requests to this controller?